scripts/20230519_add_target_account_id/script.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+
+// BEFORE
+// Count total number of payments with no received webhooks. This should be 0.
+db.getCollection('payments').countDocuments({ webhookLog: { $size: 0 } }) === 0
+
+// Count total number of payments.
+const total = db.getCollection('payments').countDocuments({})
+
+// Count total number of payments with no targetAccountId value. This should be equal to the total number of payments.
+db.getCollection('payments').countDocuments({ targetAccountId: { $exists: false } }) === total
+
+// UPDATE
+db.getCollection('payments').update(
+  {},
+  { $set: { targetAccountId: { $first: '$webhookLog.account' } } },
```

Comment:
```sh
Uncaught MongoServerError: The dollar ($) prefixed field '$first' in 'targetAccountId.$first' is not valid for storage.
```
Not sure if it's just my end - Studio3T throws this when I try to run this line?

Reply:
oooo. yup, fixed in [6f05a50](https://github.com/opengovsg/FormSG/pull/6360/commits/6f05a50b60029d851d39951de780bb02a25fba4d)!